lib/src/world.dart

```diff
@@ -17,29 +17,47 @@ class World {
     systemManager = SystemManager(this);
   }
 
+  /// Store extra data.
   void store<T>(String key, T item) => _storedItems[key] = item;
 
+  /// Retrieve extra data.
   T retrieve<T>(String key) => _storedItems[key];
 
+  /// Remove extra data.
   void remove(String key) => _storedItems[key] = null;
 
   @mustCallSuper
+
```

Comment:
Nit, I would add the annotation after the comment

Reply:
Adding the annotation after the comment will confused IDEs and you won't see the comment if you look at the method(tested in VScode)

Reply:
That is why I use vim....hahahahah